README.md

```diff
@@ -32,6 +32,33 @@
 [<img src="https://raw.githubusercontent.com/vaadin/vaadin-element/master/screenshot.png" width="200" alt="Screenshot of vaadin-element">](https://vaadin.com/elements/-/element/vaadin-element)
 
 
+## Getting Started
+
+Vaadin Elements now use the Valo theme by default. You can customize the theme variant by setting the `theme` attribute on the `:host` element. For example:
```

Comment:
Avoid using phrasing that will be outdated soon (“now use”). Just describe the current behavior like it would have always been like that.

Not sure if it’s beneficial to mention the theme variants here at all. Those are always theme-specific, and I think those are better documented in the examples/demos directly rather than in the readme.